Project/rag_engine.py

```
@@ -316,17 +336,251 @@ def chat(query, profile=None, digest=None, top_items=None, clusters=None):
+  priority: "critical" | "urgent" | "normal"
+  title: max 8 words, specific and actionable
+  body: 2-3 sentences referencing real data above
+"""
+        final_response = client.models.generate_content(model='gemini-2.0-flash', contents=final_prompt).text.strip()
+
+        if "```json" in final_response:
+            final_response = final_response.split("```json")[1].split("```")[0].strip()
+
+        try:
+            suggestions = json.loads(final_response)
+        except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)